CHANGELOG.md

```diff
@@ -35,6 +35,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 	- `name` of elements of the `configurations` array
 	- `name`, `branch`, and `version` of elements under `layers` in
 	  elements of the `configurations` array
+	- `name`, `base.product.version`, and `configuration` properties of
```

Comment:
I think it might be good to specify what the "special" variable names are (e.g. `recipe`, `image`, `base`, etc.) for the user's benefit. We don't have to do that here in the changelog, but it's probably valuable to document that somewhere. I also think we probably want to enforce somewhere that those variables can't be defined in the variables file or on the command line. (e.g. can't do something like `--vars base="something"`)

Reply:
Agree. This is a gap. I filed [CRAYSAT-1530](https://jira-pro.its.hpecorp.net:8443/browse/CRAYSAT-1530) for this.

FWIW, I think if you did try `--vars base="something"`, it would just be ignored in cases where `base` has a special meaning, i.e. in image names. I'll try to be clear about that in docs.